README.md

```diff
@@ -97,53 +180,110 @@ Create a [`views` directory](/server/views) to store server-side templates.
 $ mkdir server/views
 ```
 
-Add [server-side templating configurations to `server.js`](/server/server.js#L11-L20).
-
 Create [`index.ejs` in the views directory](/server/views/index.ejs).
 
-[Configure `server.js`](/server/server.js#L11-L20) to use server-side
+[Configure `server.js`](/server/server.js#L5-L11) to use server-side
 templating. Remember to import the [`path`](/server/server.js#L3) package.
 
+### Relpace root
+
+Replace the default `root.js` with [a new one](/server/boot/root.js) which passes a given customer's id to template engine.
+
 ### Add sample data
 
-Create three boot scripts:
+Create six boot scripts:
 
 - [`sample-customers.js`](/server/boot/sample-customers.js)
 - [`sample-orders.js`](/server/boot/sample-orders.js)
```

Comment:
Add z- in front of these to make them consistent with your changes. Also add a note as to why we are adding `z-`:

> We add `z-` in front of the boot script names to make sure they load last since LoopBack boot loads boot scripts alphabetically.